vscode-antimony/src/server/bioservices_server/webservices.py

```diff
@@ -41,6 +49,20 @@ def init_uniprot(self):
                 self.uniprot = UniProt()
             except Exception:

```

Comment:
This is an unqualified exception. Do you want to limit this to a particular type?

Reply:
This code was part of the bioservices package's code and not written by us.